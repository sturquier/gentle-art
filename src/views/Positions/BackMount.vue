<template>
  <h1>Back mount</h1>
</template>
