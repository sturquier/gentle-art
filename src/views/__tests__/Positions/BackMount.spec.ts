import { describe, it, expect } from 'vitest'
import { mount } from '@vue/test-utils'

import BackMount from '../../Positions/BackMount/BackMount.vue'

describe('BackMount', () => {
  it('renders properly', () => {
    const wrapper = mount(BackMount)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
