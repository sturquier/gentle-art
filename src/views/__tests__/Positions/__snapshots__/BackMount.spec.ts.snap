// Vitest Snapshot v1

exports[`BackMount > renders properly 1`] = `"<h1>Back mount</h1>"`;
